src/ui/foundation/types.ts

```diff
@@ -0,0 +1,23 @@
+import { Command } from './SettingsProvider/types';
+
+export class CommandEventTarget extends EventTarget {
```

Comment:
Can we move the contents of this file to another file with a more meaningful file name?